src/widget.js

```diff
@@ -161,7 +161,7 @@ Widget.prototype = {
       }
 
       let currentStateClass = this.rsWidget.className.match(/rs-state-(.+)(\s|$)/)[0];
-      this.rsWidget.classList.remove(currentStateClass);
+      this.rsWidget.classList.remove(...currentStateClass.split(' '));
```

Comment:
See above, but also this would remove ALL classes in the wrong list (it shouldn't be a list here).